perf/create-secret-manager-secret.yaml

```diff
@@ -0,0 +1,16 @@
+---
+secrets:
+  - secretName: jade-sa
+    vals:
+      - kubeSecretKey: datareposerviceaccount
+        path: secret/dsde/datarepo/perf/datarepo-sql-sa
+        encoding: base64
+        vaultKey: key
+  - secretName: database-pwd-perf
+    vals:
+      - kubeSecretKey: datarepopassword
+        path: secret/dsde/datarepo/perf/datarepo-sql-db
+        vaultKey: stairwaypassword
```

Comment:
Should this be datarepopassword instead?

Reply:
so the key was the second one cause we only created one user, it's a slight bug